internal/provider/clickhouse_cluster_resource_test.go

```diff
@@ -227,9 +244,6 @@ func sweepClickhouses(_ string) error {
 }
 
 func sweepClickhouse(conf *Config, t *clickhouse.Cluster) error {
-	op, err := conf.sdk.WrapOperation(conf.sdk.ClickHouse().Cluster().Delete(conf.ctx, &clickhouse.DeleteClusterRequest{ClusterId: t.Id}))
-	if err != nil {
-		return err
-	}
-	return op.Wait(conf.ctx)
+	_, err := conf.sdk.ClickHouse().Cluster().Delete(conf.ctx, &clickhouse.DeleteClusterRequest{ClusterId: t.Id})
```

Comment:
Wow, why do you hide operation wait?
I know It could "speedup tests" a little bit, but still we need to wait to see that all clusters deleted and we don't conflict with another runs.